openmetadata-ui/src/main/resources/ui/src/utils/GlobalSettingsUtils.tsx

```diff
@@ -225,6 +227,7 @@ export const getGlobalSettingsMenuWithPermission = (
           icon: (
             <ElasticSearchIcon className="tw-w-4 tw-mt-1.5 side-panel-icons" />
           ),
+          isAdmin: isAdminUser,
```

Comment:
We have `isProtected` flag so can we use that?

Reply:
Done